runtime/compiler/control/CompilationThread.cpp

```diff
@@ -92,7 +92,15 @@
 #include "env/J9SegmentCache.hpp"
 #include "env/SystemSegmentProvider.hpp"
 #include "env/DebugSegmentProvider.hpp"
-
+#ifdef COMPRESS_AOT_DATA
+#ifdef J9ZOS390
+#pragma convlit(suspend)
+#include <zlib.h>
+#pragma convlit(resume)
```

Comment:
Added comment in https://github.com/eclipse/openj9/pull/5926/commits/295afedd848d1b5a1a79de78788a9b4d85b3e30a

Reply:
The comment added in 295afed does not explain _why_ we need the `#pragma`. It simply explains what the `#pragma` will do which is obvious if one understands `convlit`. I believe the reason _why_ we need to avoid literal conversion was because the version string encoded as a literal string in `zlib.h` and some version check was failing when it tried to compare that string. I forget the details, but I think you recall better.

Reply:
Yeah, it was because different string encoding between statically available zlib binary and our codebase. Added comment : https://github.com/eclipse/openj9/pull/5926/commits/c104a31c9b3257335036852a09e754ddaf29e5a9

Reply:
Excellent, the explanation is very clear now. Thanks for making the change!